specs/commandline/sign.md

```diff
@@ -162,10 +163,43 @@ Successfully signed localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da
 
 ### [Experimental] Sign an artifact and store the signature using OCI artifact manifest
```

Comment:
Are we putting `--signature-manifest` behind experimental flag?

Reply:
@priteshbandi  Yes, it is experimental indeed, as you may know that artifact manifest is removed from OCI 1.1 scope.